perf/lua/1mops_write.lua

```diff
@@ -11,7 +11,9 @@ local math = require('math')
 local json = require('json')
 local fio = require('fio')
 
-local params = require('internal.argparse').parse(arg, {
+local benchmark = require('benchmark')
```

Comment:
Can we make it optional, so that standalone 1mops still can be used?

Reply:
It can be run standalone, like the following:
```sh
LUA_PATH="perf/lua/?.lua;;" src/tarantool perf/lua/1mops_write.lua 
# making 10000000 REPLACE operations,
# 150 operations per txn,
# using 50 fibers,
# in a replicaset of 1 nodes,
# using HASH index type
# with WAL mode write
# 
# Warmup... done, lsn:  1006207
# master done 8991300 ops in time: 16.221424, cpu: 21.494186
# master average speed  554285  ops/sec
# master peak speed     664726  ops/sec
1mops_master 554285 rps
```
Do you have concerns about the `LUA_PATH` declaration?

Reply:
It is always required since we always have common parts for formatting (even in console mode) and timing aggregation (moreover, timing counting may be moved to this module later, when we will have a clear vision of the API).

Reply:
I've added stubs for all places where it is necessary, so 1mops can still be run standalone.